addons/binding/org.openhab.binding.geofence/ESH-INF/config/config.xml

```diff
@@ -0,0 +1,86 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<config-description:config-descriptions
+		xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+		xmlns:config-description="http://eclipse.org/smarthome/schemas/config-description/v1.0.0"
+		xsi:schemaLocation="http://eclipse.org/smarthome/schemas/config-description/v1.0.0
+		http://eclipse.org/smarthome/schemas/config-description-1.0.0.xsd">
+	<config-description uri="thing-type:geofence:device">
```

Comment:
This is missing the "trackerId" parameter - this should be the only mandatory parameter as it is required to identify the thing.

Reply:
Yes, it is mandatory, but it's coming from the GPS records and it shouldn't be changed. Adding tracker id as parameter will make it writable on OH side and I'm not sure we should allow that. Allowing edit on two points can lead to misconfiguration.

Reply:
No, you are mistaking the intention of config parameters. They configure the HANDLER, so that the handler knows what tracker it is associated to. Therefore you must provide this information as a configuration parameter to it.

Reply:
But trackerId = thing id. Are you saying that the thing id should be duplicated in the configuration? The thing id is persistent, isn't it?

Reply:
The ThingUID must not carry any semantics - it can be chosen freely by ThingProviders.

Reply:
I was not aware of this constraint. I added the trackerId parameter and removed all thing.getUID().getId() calls.